data/alpha/field/internal-drainage-board.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: internal-drainage-board
+phase: alpha
+register: ''
```

Comment:
This needs a register property of `internal-drainage-board` since this field always belongs to that register